.github/workflows/release-pr.yml

```diff
@@ -58,76 +83,26 @@ jobs:
       tag: ${{ needs.get-pr-number.outputs.pr-number }}
       dockerfile_path: images/openscap/Dockerfile
       vendor: 'Compliance Operator Authors'
+      checkout_ref: refs/pull/${{ needs.get-pr-number.outputs.pr-number }}/head
 
   catalog-container-push-pr:
-    runs-on: ubuntu-latest
     needs:
     - get-pr-number
     - bundle-container-push-pr
-    if: always() # remove this after we fix signing
-
     permissions:
-      packages: write
       contents: read
-
-    outputs:
-      image-digest: ${{ steps.container_info.outputs.image-digest }}
-      image-tags: ${{ steps.container_info.outputs.image-tags }}
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3.6.0
-
-      - name: Login to ghcr.io
-        uses: docker/login-action@v3.0.0
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-
-      - name: Docker metadata
-        id: meta
-        uses: docker/metadata-action@v4
-        with:
-          images: ghcr.io/complianceascode/compliance-operator-catalog
-          tags: |
-            type=raw,value=${{ needs.get-pr-number.outputs.pr-number }}
-            type=sha,format=long
-          labels: |
-            org.opencontainers.image.source=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}
-            org.opencontainers.image.title=compliance-operator-catalog
-            org.opencontainers.image.version=latest
-            org.opencontainers.image.licenses='Apache-2.0'
-            org.opencontainers.image.vendor='Compliance Operator Authors'
-        
-      - name: Generate catalog dockerfile
-        id: generate_catalog_dockerfile
-        run: |
-          make catalog-docker BUNDLE_IMGS=ghcr.io/complianceascode/compliance-operator-bundle:${{ needs.get-pr-number.outputs.pr-number }}
-
-      - name: Build container images and push
-        id: docker_build
-        uses: docker/build-push-action@v5
-        with:
-          context: .
-          file: catalog.Dockerfile
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          push: true
-          platforms: 'linux/amd64'
-
-      - name: Get container info
-        id: container_info
-        run: |
-          image_tags="${{ needs.get-pr-number.outputs.pr-number }}"
-          echo "::set-output name=image-digest::${{ steps.docker_build.outputs.digest }}"
-          echo "::set-output name=image-tags::${image_tags}"
+      id-token: write
+      packages: write
+    uses: metal-toolbox/container-push/.github/workflows/container-push.yml@main
```

Comment:
Didn't we run into issues with `metal-toolbox/container-push` and `workflow_run`?
https://github.com/ComplianceAsCode/content/pull/11533

Reply:
I had a fix in https://github.com/metal-toolbox/container-push/pull/68